Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1113,7 +1113,7 @@ class BrowserViewController: UIViewController {
             return
         }
 
-        if let url = webView.url {
+        if tab !== tabManager.selectedTab, let url = webView.url {
```

Comment:
The first patch I did for this missed the  `tab !== tabManager.selectedTab, ` and the code in this block is only for background tabs.